package.json

```diff
@@ -45,7 +46,11 @@
     "eslint-watch": "^2.1.13",
     "flux-standard-action": "^0.6.0",
     "mocha": "^2.2.5",
-    "rimraf": "^2.5.3"
+    "rimraf": "^2.5.3",
+    "flux-standard-action": "^0.6.0",
+    "lodash.isplainobject": "^3.2.0",
```

Comment:
Is there a reason `lodash.isplainobject` is being added back?

See https://github.com/acdlite/redux-actions/pull/107#discussion_r70169615.